strings/redpill.json

```diff
@@ -0,0 +1,19 @@
+{
+	"redpill_questions": [
+		"What happened to the ceiling?",
+		"Why is it called the emergency shuttle if it arrives every single shift?",
+		"Where does the Cook get all this meat from?",
+		"Space wind? How does that even make sense?",
+		"Why does Nanotrasen hire Clowns and Mimes for every single station?",
+		"Why is the station's air supply connected to the plasma tank?",
+		"Why are there fire alarms everywhere but no sprinklers?",
+		"Why is this a plasma research station if we know everything about plasma already?",
+		"Who hires all these unqualified, violent assistants?",
+		"How do the morgue trays know whether someone's soul is still in their body?",
+		"If two miners were in Lavaland and one of them killed the other with a pickaxe, wouldn't that be fucked up or what?",
```

Comment:
***would** that be fucked up or what?

 "If there were two guys on the moon and one of them killed the other with a rock would that be fucked up or what?"